src/drivers/ms5611/ms5611_posix.cpp

```diff
@@ -976,6 +976,7 @@ struct ms5611_bus_option &find_bus(enum MS5611_BUS busid)
 	}	
 	// FIXME - This is fatal to all threads
 	errx(1, "bus %u not started", (unsigned)busid);
+	return bus_options[0];
```

Comment:
@LorenzMeier Is this the right way to make the compiler stop complaining about "reached end of non volatile function"?